soccer/src/soccer/strategy/agent/position/offense.cpp

```diff
@@ -65,16 +61,16 @@ Offense::State Offense::update_state() {
             }
             break;
         case STEALING:
-            // transition to idling if we are close enough to the ball
-            if (check_is_done()) {
-                /* SPDLOG_INFO("\033[92m ball pos {}{} \033[0m", ball_position.x(),
-                 * ball_position.y()); */
-
+            // The collect planner check_is_done() is wonky so I added a second clause to check
```

Comment:
is there a way to fix that planner's is_done? (e.g. just add the second clause into that is_done directly)